wicket-console/pom.xml

```diff
@@ -35,6 +35,47 @@
 	    <artifactId>wicket-devutils</artifactId>
 	    <scope>provided</scope>
     </dependency>
+    		<dependency>
```

Comment:
Why so big set of dependencies? I didn't expect any new dependencies for just swithcing to button.